src/translations/en.json

```diff
@@ -2033,26 +2040,33 @@
               "geo_location_sources": "Geolocation Sources",
               "dark_mode": "Dark Mode?",
               "default_zoom": "Default Zoom",
-              "source": "Source"
+              "source": "Source",
+              "description": "A card that allows you to display entities on a map."
             },
             "markdown": {
               "name": "Markdown",
-              "content": "Content"
+              "content": "Content",
+              "description": "Markdown card is used to render Markdown."
             },
             "media-control": {
-              "name": "Media Control"
+              "name": "Media Control",
+              "description": "The media control card is used to display Media Player entities on an interface with easy to use controls."
```

Comment:
The name of the card isn't capitalized in this description, but is in most others. Which is the correct way of naming it?

Reply:
I believe they should be capital